workflow-report/action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Report the failed job to VK Team chat'
+description: 'This action creates a message about failed job and sends it to VK Teams chat'
+
+inputs:
+  api-url:
+    description: 'Bot API URL'
+    required: true
+  bot-token:
+    description: 'Bot token'
+    required: true
+  chat-id:
+    description: 'Notification chat ID (or stamp from chat URL)'
+    required: true
+  job-steps:
+    description: 'Steps info from github context'
```

Comment:
Info about steps that have been run in the job

Reply:
done